datalad/distribution/get.py

```diff
@@ -237,6 +237,13 @@ def _get_flexible_source_candidates_for_submodule(ds, sm):
                         ds_repo.config[c])
                        for c in ds_repo.config.keys()
                        if c.startswith(candcfg_prefix)]:
+        # ensure that there is only one template of the same name
+        if type(tmpl) == tuple and len(tmpl) > 1:
+            raise ValueError(
+                f"There are multiple URL templates for submodule clone "
+                f"candidate '{name}', but only one is allowed. "
+                f"Check your configuration!"
```

Comment:
```suggestion
                f"Check datalad.get.subdataset-source-candidate-* configuration!"
```

So users have an immediate idea what to look for.

Reply:
that makes sense, thx for the suggestion

Reply:
Thank you for noticing this! I meant to commit it, and thought I did, but it must have gotten lost somewhere - I still had the commit message for it cached

Reply:
it should be applied now